setuptools/_core_metadata.py

```diff
@@ -88,6 +88,7 @@ def read_pkg_file(self, file):
     self.url = _read_field_from_msg(msg, 'home-page')
     self.download_url = _read_field_from_msg(msg, 'download-url')
     self.license = _read_field_unescaped_from_msg(msg, 'license')
+    self.license_expression = _read_field_unescaped_from_msg(msg, 'license_expression')
```

Comment:
Should this use an `_` (underscore) or `-` (dash) for the message field string?

Reply:
Yes, it should be `-`. Will update it.